api/product.rb

```diff
@@ -45,7 +45,7 @@ class Version < Api::Object
       attr_reader :default
       attr_reader :name
 
-      ORDER = %w[v1 beta alpha].freeze
+      ORDER = %w[v2 v1 beta alpha].freeze
```

Comment:
I've been meaning to change 'v1' to 'ga' for a while now. In the meantime, can you leave this as-is and just label the version as v1 in api.yaml?

Reply:
Done.